src/integrationTest/java/uk/gov/hmcts/reform/pdf/service/integration/GeneratedPDFContentV2Test.java

```diff
@@ -56,9 +55,8 @@ private static String textContentOf(byte[] pdfData) throws IOException {
         }
     }
 
-    @SuppressWarnings("unchecked")
     @Test
-    @Category(SmokeTest.class)
```

Comment:
So there won’t be any smoke tests ? Or is added/exists separately ?

Reply:
Moved from integration tests which didn't run a real test to here:
https://github.com/hmcts/cmc-pdf-service/pull/72/files#diff-1b3ae1409a16ef8bfcc0282291571943R39